src/client/kinto_http.rs

```diff
@@ -355,10 +412,9 @@ mod tests {
             }
             e => assert!(false, "Unexpected error type: {:?}", e),
         };
-
-        get_changeset_mock.delete();
     }
 
+    /* // TODO: This should really be in the HTTP tests
```

Comment:
Please open an issue to keep track of it

Reply:
I ended up re-enabling it :)